WalletWasabi.Backend/WalletWasabi.Backend.csproj

```diff
@@ -2,6 +2,8 @@
 
 	<PropertyGroup>
 		<DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
+		<AnalysisLevel>latest</AnalysisLevel>
```

Comment:
These properties don't belong to here, they are defined at root directory level. The same applies to all csproj files

Reply:
OK, removed! Thanks :)